pkg/sfu/streamallocator.go

```diff
@@ -1514,6 +1574,12 @@ func (c *ChannelObserver) AddEstimate(estimate int64) {
 }
 
 func (c *ChannelObserver) AddNack(packets uint32, repeatedNacks uint32) {
+	if c.params.NackWindowDuration != 0 && time.Since(c.nackWindowStartTime) > c.params.NackWindowDuration {
+		c.nackWindowStartTime = time.Now()
+		c.packets = 0
+		c.repeatedNacks = 0
+	}
```

Comment:
Checking NACKs in the most recent window only. In probe, this window duration is 0. So, it is checking for all of probe. In probe, we want to stop as soon as possible on a failure. I think windowing there also might be better. Need to test more. Keeping it as before for now. Will be testing more with a different window duration during probe also.